src/os/bluestore/bluestore_types.h

```diff
@@ -156,7 +156,8 @@ struct denc_traits<vector<bluestore_pextent_t>> {
   static void bound_encode(const vector<bluestore_pextent_t>& v, size_t& p) {
     p += sizeof(uint32_t);
     size_t per = 0;
-    denc(*(bluestore_pextent_t*)nullptr, per);
+    bluestore_pextent_t tmp;
```

Comment:
thanks for you insights, @adamemerson ! maybe we can wait for the opinions from @liewegas and @allensamuels .